ols/src/docs/docs_summarizer.py

```diff
@@ -72,16 +73,21 @@ def summarize(self, conversation: str, query: str, **kwargs) -> tuple[str, str]:
         )
 
         # TODO get this from global config
-        storage_context = StorageContext.from_defaults(
-            persist_dir=constants.PRODUCT_DOCS_PERSIST_DIR
-        )
-        self.logger.info(f"{conversation} Setting up index")
-        index = load_index_from_storage(
-            storage_context=storage_context,
-            index_id=constants.PRODUCT_INDEX,
-            service_context=service_context,
-            verbose=verbose,
-        )
+        try:
+            storage_context = StorageContext.from_defaults(
+                persist_dir=constants.PRODUCT_DOCS_PERSIST_DIR
+            )
+            self.logger.info(f"{conversation} Setting up index")
+            index = load_index_from_storage(
+                storage_context=storage_context,
+                index_id=constants.PRODUCT_INDEX,
+                service_context=service_context,
+                verbose=verbose,
+            )
+        except FileNotFoundError as err:
+            self.logger.error(f"FileNotFoundError: {err.strerror}, file= {err.filename}")
+            self.logger.info("Initializing empty vector index.")
+            index = VectorStoreIndex([])
```

Comment:
discussed on scrum, if we have no vector index we should not setup a query engine, and we also need to send back additional information in the response object indicating that the response was generated w/o access to RAG content.